build/downloader/src/cli.ts

```diff
@@ -136,7 +137,15 @@ export const installFile = async (toolName: string, common: CommonArgs, toolVers
 
 export const useFile = async (toolName: string, version: string, common: CommonArgs) => {
   const toolFolder = path.resolve(common.appData, 'tools', toolName)
-  const underscoreVersion = version.replace(/\./g, '_')
+
+  const parsed = semverParse(version)
+  if (!parsed) {
+    return logger.error('Invalid semver version.')
+  }
+
+  const { major, minor, patch, prerelease } = parsed
+  const underscoreVersion = [major, minor, patch, ...prerelease].join('_')
```

Comment:
Please remove the semver checking, it will not work with my upcoming changes to the downloader (versions can be branchn names)